fossology/fossology
# feat(copyright): Use package based dependency for json

## Description

Remove dependency from external file download for json-hpp and shift to packaged based dependency (`libjsoncpp` from Debian based and `jsoncpp` from RedHat based systems).

### Changes

1. Remove download for external json-hpp file.
1. Add `jsoncpp` package dependency.

## How to test

Compile the `copyright` package in Debian and RedHat based systems and check the JSON output.

```bash
$ sudo ./utils/fo-installdeps -ye
$ make copyright
$ cd src/copyright/agent
$ ./copyright -J ./copyright.cc
```

Closes #1084
